docs/index.md

```diff
@@ -0,0 +1,26 @@
+# Home
+
+This is a documentation of all OTCamera modules and configuration variables.
+OTCamera is part of the [OpenTrafficCam](https://opentrafficcam.org) project and is used to record videos of traffic over several hours or even days.
+
+On the left side you can see the main module [record](record) along with the [config](config) and [status](status) module, which contain variables used across the entire code.
+
+It is build completly automatically using:
+
+- [MkDocs](https://www.mkdocs.org/) and
+- [mkdocstrings](https://github.com/pawamoy/mkdocstrings) and
+- [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) theme.
+
+To work on the docs run the following command to get a live updating server. You have to run it on the [OTCamera dev pi](https://opentrafficcam.org/contribute/otcamera).

```

Comment:
Ok so I can´t work on the docs by just changing or creating a md file? Maybe this is something you can show to us on an OTFriday ;)

Reply:
Sure you can. Beside the md files containing the `::: somemodule` you can add as many manually written stuff.